.devcontainer/devcontainer.json

```diff
@@ -6,7 +6,7 @@
   "postStartCommand": "script/bootstrap",
   "containerEnv": { "DEVCONTAINER": "1" },
   "appPort": 8123,
-  "runArgs": ["-e", "GIT_EDITOR=code --wait"],
+  "runArgs": ["-e", "GIT_EDITOR=code --wait", "--network=host"]
```

Comment:
This change is not related and should not be added. Please revert this.

Reply:
@frenck will do, i am at a loss how to use the devcontainer process, suggested earlier, to run a dev instance that binds to the network host - without that i can't test the changes in my PR because DHCP packets won't be seen by processes in the container will they?

I left my first devcontainer running for an hour and it detected nothing on my network whatsoever.

Reply:
The dev container does not bind to the host network, as the host network is not supported by all operating systems (e.g., MacOS, Windows is problematic as well).

Reply:
ok, that makes sense to me, i only did it because i was prompted to
i will go create a dev image and use macvlan on a linux host, or just use a linux vm.  
thanks